docs/stats.rst

```diff
@@ -0,0 +1,8 @@
+.. _stats:
+
+=====
+Stats
+=====
+
+scrapy-poet tracks :external+web-poet:ref:`web-poet stats <stats>` as part of
+:ref:`Scrapy stats <topics-stats>`, prefixed with ``poet/stats/``.
```

Comment:
Last thing to check: let's see how it looks like with "standard stats", like the number of dependencies provided:

```
"poet/stats/my-custom-1": 15,
"poet/stats/my-custom-2": 15,
"poet/dependencies/MyPageObject": 15,
"poet/dependencies/HttpResponse": 15,
```

Seems fine to me, so +1 to merge